test.js

```diff
@@ -219,6 +219,7 @@ test('JIRA to Slack: Check All Formatting', (assert) => {
     'Multiple Links: [Someurl1|http://someurl1.com] links to [Someurl2|http://someurl2.com]\n' +
     'Blockquote: \nbq. This is quoted\n' +
     'Color: {color:white}This is white text{color}\n' +
+    'Multiple Colors: {color:white}This is white text{color} {color:black}This is black text{color}\n' +
```

Comment:
bonus, since this is all one big block of text, just by duplicating the previous line we could make the old code fail, but this makes it explicit what is being tested.

Reply:
Looks good. I'll probably add an explicit test for this too.

Reply:
nice, thanks!